cmd/session.go

```diff
@@ -354,3 +327,69 @@ func SessionTerminate(cmd *cobra.Command, args []string) error {
 	ui.Successf("Session terminated")
 	return nil
 }
+
+// sessionSelectSSHExecRef selects an exec id from all sessions in the Unweave environment or whether to create a new
+// provides an option to create a new Exec an error or exits if unrecoverable
+func sessionSelectSSHExecRef(cmd *cobra.Command, execRef string, allowNew bool) (string, bool, error) {
+	const newSessionOpt = "create a new session"
+
+	execs, err := getExecs(cmd.Context())
+	if err != nil {
+		if e, ok := err.(*types.Error); ok {
+			uie := &ui.Error{Error: e}
+			fmt.Println(uie.Verbose())
+			os.Exit(1)
+		}
+		return "", false, err
+	}
+
+	opts, execIdByOpts := formatExecCobraOpts(execs)
+
+	if !allowNew {
+		opts = append(opts, newSessionOpt)
+		execIdByOpts[len(opts)-1] = newSessionOpt
+	}
```

Comment:
Can we add the option at the top of the list

Reply:
Implemented